components/brave_wallet/renderer/js_solana_provider.h

```diff
@@ -32,16 +32,6 @@ class JSSolanaProvider final : public gin::Wrappable<JSSolanaProvider>,
 
   static gin::WrapperInfo kWrapperInfo;
 
-  class V8ConverterStrategy : public content::V8ValueConverter::Strategy {
```

Comment:
Why do we remove it entirely?
Have you tried `solana.publicKey` and `solana.signTransaction` after removal?

Reply:
This
https://github.com/brave/brave-core/pull/13419/files#diff-7cdd1da2bab576a68d9a1469b45add340ad978aafbdff369f3e590b1aaf8febb

changed v8array -> base::Value conversion from Value(base58 string) to Value(blob) which is exactly what default implementation does https://source.chromium.org/chromium/chromium/src/+/main:content/renderer/v8_value_converter_impl.cc;l=503-529
so we don't need custom conversion strategy

Tried `solana.publicKey` and it works as before